app/modules/encounters/models.py

```diff
@@ -147,15 +147,23 @@ def delete_from_edm(self, current_app):
     # given edm_json (verbose json from edm) will populate with houston-specific data from feather object
     # note: this modifies the passed in edm_json, so not sure how legit that is?
```

Comment:
Legit in what what way? Seems perfectly valid to me and actually a really neat solution of how we tweak the responses received from EDM. 
The only thing I'm thinking is that we could potentially require this to update a json response from WBIA as well as EDM so maybe augment_json would be a better name. 
I seem to have missed this PR previously.

Reply:
i was mostly referring to "legit" in terms of pythonic okayness.  meaning: i am modifying the _passed in value_ (which i will assume is passed-by-reference) and thus tinkering with the value the caller had passed in.  in which case, why would i then need to _return_ it from this method?   basically my python newbieness bumping up against unknown practices, being expressed as a comment.  haha